.circleci/config.yml

```diff
@@ -407,6 +407,15 @@ jobs:
       - run: docker run --privileged linuxkit/binfmt:v0.6
       - run: cp dist/grafana-latest.linux-*.tar.gz packaging/docker
       - run: cd packaging/docker && ./build.sh --fast "${CIRCLE_SHA1}"
+      - run:
+          name: Archive Docker image
```

Comment:
Why do we need to store the Docker image as an artifact, and where is the artifact used?

Reply:
For debug/test purpose. Think these changes are quite easy to debug/test locally since you can use the latest grafana 6.4.4 tarball to build a local docker image.

Reply:
i wanted to make sure circleci was producing what was expected and not my local machine.  turns out this helped to troubleshoot the problem. we can remove it though, i'll keep it around for future debugging vs having this be done for every build.

Reply:
latest commit removes this